src/profile-logic/import/instruments/BinReader.js

```diff
@@ -0,0 +1,55 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+// @flow
+
+class BinReader {
```

Comment:
It's good to add a comment for this and mention where this class is from.

Reply:
Also I think if we can find a well maintained npm package that does this work would be better, not sure if there is though.